tests/test_cache_policies.py

```diff
@@ -40,26 +40,6 @@ def test_addition_of_none_is_noop(self, typ):
         assert policy + other == other
 
 
-class TestDefaultPolicy:
```

Comment:
I think it'd be worthwhile to keep this test class and update it to cover the new behavior of the default cache policy.

Reply:
Will do!